src/components/domain/BeaconUseSection.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { FunctionComponent } from "react";
+import { DraftBeaconUse } from "../../entities/DraftBeaconUse";
 import { BeaconUse } from "../../lib/registration/types";
 import { ordinal, prettyUseName, sentenceCase } from "../../lib/writingStyle";
 import { SummaryList, SummaryListItem } from "../SummaryList";
 import { AnchorLink, SectionHeading, WarningLink } from "../Typography";
 
 interface BeaconUseSectionProps {
   index: number;
-  use: BeaconUse;
+  use: DraftBeaconUse;
```

Comment:
Capture the concept of a use for a beacon that the user is editing as a `DraftBeaconUse`.  This will become more useful when we are using the same form flows to edit an existing `BeaconUse`.